src/enum.cr

```diff
@@ -276,6 +276,21 @@ struct Enum
     value.hash
   end
 
+  def each
+    {% if @type.has_attribute?("Flags") %}
+      return if value == 0
+      {% for member in @type.constants %}
+        {% if member.stringify != "All" %}
+          if includes?({{@type}}::{{member}})
+            yield {{@type}}::{{member}}
+          end
+        {% end %}
+      {% end %}
+    {% else %}
+      {% raise "Can't iterate #{@type}: only Flags Enum can be iterated" %}
```

Comment:
Oooooh... I see. Yeah, I thought it was `Enum.each` 😊 Thanks for the explanation!